src/swagger-specs/reactor.yaml

```diff
@@ -9385,7 +9388,7 @@ definitions:
           property: https://reactor.adobe.io/secrets/SE39fdf431f8ad4600bbc24ea73fcb1154/property
         meta:
           test_exchange:
-            access_token: FpIkBn3zxW0fH6EBC4MB
+            access_token: some-access-token
```

Comment:
I'm guessing this is part of the user study feedback. It is odd to replace the hard value example with a non-exemplary placeholder, but if that is feedback from users then that's fine